eunkyung/.gitignore

```
@@ -0,0 +1,2 @@
+node_modules
+.env
```

db 폴더 안에 schema.sql 파일은 로컬 환경에 따라 다르게 적용될 수 있습니다.

migration에 대한 이력 자체는 migrations/*.sql 파일들로 충분히 확인할 수 있기 때문에 schema.sql 파일은 GitHub 상에서 공유되지 않도록 수정해주세요.

이미 GitHub에 push 된 내역을 지우기 위해서는 아래와 같은 명령어를 사용할 수 있습니다:
`git rm --cached "file-path"`
```suggestion
node_modules
.env
db/schema.sql
```